parquet-avro/src/test/java/org/apache/parquet/avro/TestAvroSchemaConverter.java

```diff
@@ -128,6 +216,9 @@ public void testAllTypes() throws Exception {
             "  optional group myoptionalarray (LIST) {\n" +
             "    repeated int32 array;\n" +
             "  }\n" +
+            "  required group myarrayofoptional (LIST) {\n" +
+            "    repeated int32 array;\n" +
```

Comment:
myarrayofoptional with "repeated int32"? With repeated, elements will be optional?